Readme.md

```diff
@@ -82,7 +82,9 @@ dateFormat(now, "N");
 | `d`              | Day of the month as digits; no leading zero for single-digit days.                                                                                            |
 | `dd`             | Day of the month as digits; leading zero for single-digit days.                                                                                               |
 | `ddd`            | Day of the week as a three-letter abbreviation.                                                                                                               |
+| `DDD`            | "Yesterday", "Today" or "Tomorrow" if date lies within these three days. Else fall back to ddd.                                                               |
```

Comment:
Please can we change the `DDD` Mask to show `Yes`, `Tod` & `Tom`. This is because the `ddd` mask which it defaults to always shows 3 characters. And so the user would likely be designing their UI around expecting three characters.